governance/voting-committees/dvc-1/strategies.json

```diff
@@ -0,0 +1,8 @@
+{
+    "risk": [
+        "0x7Ddb50A5b15AeA7e7cf9aC8E55a7F9fd9d05ecc6"
+    ],
```

Comment:
Based on our meeting the other week, I think strategies won't have a single contract as shown above. They will have a name, picture (so profile, I guess), and then a link to contracts that are a combination of Delegate and Strategy:

```
Strategy = {
    name,
    picture,
    ...,
    contracts: [{ DelegateStrategyContract }]
}
```